src/sentry/uptime/endpoints/organization_uptime_stats.py

```diff
@@ -0,0 +1,175 @@
+import logging
+from collections import defaultdict
+
+from drf_spectacular.utils import extend_schema
+from google.protobuf.timestamp_pb2 import Timestamp
+from rest_framework.request import Request
+from rest_framework.response import Response
+from sentry_protos.snuba.v1.endpoint_time_series_pb2 import TimeSeriesRequest, TimeSeriesResponse
+from sentry_protos.snuba.v1.request_common_pb2 import RequestMeta, TraceItemType
+from sentry_protos.snuba.v1.trace_item_attribute_pb2 import (
+    AttributeAggregation,
+    AttributeKey,
+    AttributeValue,
+    Function,
+    StrArray,
+)
+from sentry_protos.snuba.v1.trace_item_filter_pb2 import ComparisonFilter, TraceItemFilter
+
+from sentry.api.api_owners import ApiOwner
+from sentry.api.api_publish_status import ApiPublishStatus
+from sentry.api.base import StatsArgsDict, StatsMixin, region_silo_endpoint
+from sentry.api.bases.organization import OrganizationEndpoint, OrganizationPermission
+from sentry.models.organization import Organization
+from sentry.models.project import Project
+from sentry.uptime.models import ProjectUptimeSubscription, UptimeSubscription
+from sentry.utils.snuba_rpc import timeseries_rpc
+
+logger = logging.getLogger(__name__)
+
+
+@region_silo_endpoint
+@extend_schema(tags=["Crons"])
+class OrganizationUptimeStatsEndpoint(OrganizationEndpoint, StatsMixin):
+    publish_status = {
+        "GET": ApiPublishStatus.EXPERIMENTAL,
+    }
+    owner = ApiOwner.CRONS
+    permission_classes = (OrganizationPermission,)
+
+    def get(self, request: Request, organization: Organization) -> Response:
+        timerange_args = self._parse_args(request)
+        projects = self.get_projects(request, organization)
+
+        uptime_subscription_ids = request.GET.getlist("uptime_subscription_id")
```

Comment:
@wedamija These are the subscription IDs. We don't actually return these to the frontend yet in the ProjectUptimeMonitorSerializer yet, it's going to be confusing though because this name `uptime_subscription_id` is also used to refer to the subscription table in the projectsubsciptions table.

lmk if you think there could be a better name.

Reply:
This is `UptimeSubscription.id` or `ProjectUptimeSubscription.id`?

Reply:
neither. it is `UptimeSubscription.subscription_id`. do we want to change this?

Reply:
we only store `UptimeSubscription.subscription_id` in snuba, but we can map to that in this endpoint.

Reply:
Yeah I think we should translate this

Reply:
Yeah I'd say we should translate it. It's an internal id that we don't really expose to the user

Reply:
i've gone ahead and implemented translation